cms/server/admin/templates/fragments/overload_warning.html

```diff
@@ -9,7 +9,7 @@
 #}
 
 {% if config.contest_listen_port|length > 0 %}
-  {% set ratio = round(contest.participations|length * 1.0 / config.contest_listen_port|length, 2) %}
+  {% set ratio = (contest.participations|length / config.contest_listen_port|length)|round(2) %}
```

Comment:
Python 2?

Reply:
This isn't Python, it's Jinja, and in Jinja semantics `/` means floating-point division and `//` means integer division, for both Python versions. See http://jinja.pocoo.org/docs/2.10/templates/#math

Reply:
Cool, another syntactically similar but slightly semantically different language!